app/.eslintrc.cjs

```diff
@@ -2,6 +2,7 @@ const IGNORED_UNICORN_RULES = {
   "unicorn/filename-case": "off",
   "unicorn/no-null": "off",
   "unicorn/prevent-abbreviations": "off",
+  "unicorn/no-nested-ternary": "off",
```

Comment:
Denne klager ikke på nested ternaries, men at man må wrappe dem i parenteser. Men det fjerner prettier for deg, så da blir det krangling.

Jeg synes uansett det er penere med nøsta ternaries, så slo av denne.